docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,174 @@
+.. _Courses API Courses Resource:
+
+########################################
+Courses API Courses Resource
```

Comment:
This should probably live in the catalog document.  In this case the courses are the set of courses returned using the catalog's query.  There is a completely separate endpoint `/courses/` which returns all courses